protocol/contracts/libraries/Silo/LibLegacyTokenSilo.sol

```diff
@@ -505,4 +530,56 @@ library LibLegacyTokenSilo {
         }
         return 0;
     }
+
+    ////////////////////////// CLAIM ///////////////////////////////
+    // as of the zero withdraw update, these functions are no longer used. 
+    // However, these are kept for backwards compatability.
```

Comment:
Leave one line break between this and the section header. 

For functions in a contract, always use the block comment and an `@dev` natspec comment